gramps/gui/editors/editreporef.py

```diff
@@ -36,12 +36,22 @@
 _ = glocale.translation.gettext
 from gramps.gen.lib import NoteType
 from gramps.gen.db import DbTxn
+from gramps.gen.const import URL_MANUAL_SECT2
 
 from .displaytabs import NoteTab, AddrEmbedList, WebEmbedList, SourceBackRefList
 from ..widgets import MonitoredEntry, PrivacyButton, MonitoredDataType
 from .editreference import RefTab, EditReference
 from ..glade import Glade
 
+# -------------------------------------------------------------------------
+#
+# Constants
+#
+# -------------------------------------------------------------------------
+
+WIKI_HELP_PAGE = URL_MANUAL_SECT2
+WIKI_HELP_SEC = _("Repository_Reference_Editor", "manual")
```

Comment:
References to manual pages should no longer be translated.  The translation should be handled by page re-direction in the wiki instead.